packages/compas-open-scd/src/translations/en.ts

```diff
@@ -871,6 +871,10 @@ export const en = {
       server: 'Error communicating with CoMPAS Ecosystem',
       serverDetails: '{{type}}: {{message}}',
     },
+    warning: {
+      nsdoc: 'Process NSD loading error',
```

Comment:
Is it applicable to all NSD files or NSdoc files only?
Possible: "Could not load NSdoc files, not files available on the server."

Reply:
NSdoc files only. I got the title out of the issue. I think what you suggest might be a bit long for a log title. Maybe just `Could not load NSdoc file`?